### PR TITLE
Fix for crew monitoring computer and cyborgs

### LINKED ISSUE #10380: Crew monitor console does not work for borgs from a distance
The crew list is empty when you access it from a distance.
Works fine when you're standing next to it.

### PR DESCRIPTION
**Pull request by nullquery**
This fixes #10380. The crew monitor computer updates for cyborgs, but only if they remain in range of the computer.

### CODE REVIEW

File: code/game/machinery/computer/crew.dm [hunk 1]
```diff
@@ -199,7 +199,14 @@ var/global/datum/crewmonitor/crewmonitor = new
 	for (z in src.interfaces)
 		if (src.interfaces[z] == hi) break
 
-	return (hclient.client.mob && hclient.client.mob.stat == 0 && hclient.client.mob.z == text2num(z) && ( isAI(hclient.client.mob) || (locate(/obj/machinery/computer/crew, range(1, hclient.client.mob))) || (locate(/obj/item/device/sensor_device, hclient.client.mob.contents)) ) )
+	if (hclient.client.mob && hclient.client.mob.stat == 0 && hclient.client.mob.z == text2num(z))
+		if (isAI(hclient.client.mob)) return TRUE
+		else if (isrobot(hclient.client.mob)) // fix for /tg/station issue #10380 to allow borgs to operate the crew monitoring computer remotely
```
Comment (RemieRichards) [inline code/game/machinery/computer/crew.dm]: don't do this.
We don't comment on the bugs we fix with the actual issue report.
Not to mention it's clearly readable from this code that it's designed to make robots able to use it at range.

tl;dr remove the comment.
